stories/Icons.story.tsx

```diff
@@ -0,0 +1,341 @@
+import * as React from 'react';
+
+import {
+    description,
+    importExample,
+    header,
+    divider,
+    tab,
+    tabs,
+    title,
+    code as baseCode
+} from 'wix-storybook-utils/Sections';
+
+
+import { allComponents } from './utils/allComponents';
+
+const code = config =>
+    baseCode({ components: allComponents, compact: true, ...config });
+
+export default {
+    category: 'Icons',
+    storyName: 'Icons',
+    sections: [
+        header(),
+        tabs([
+            tab({
+                title: 'Icon Inventory',
+                sections: [
+                    description({
+                        title: 'Importing the icons',
+                        text: [
+                            'We are using the ',
+                            <a href="https://www.npmjs.com/package/@svgr/webpack" >
+                                svgr plugin
+                            </a>,
+                            ' for svg imports which allow you to import the svg file directly. ',
+                            'If you have ',
+                            <a href="https://webpack.js.org/guides/tree-shaking/" >
+                                Tree Shaking
+                            </a>,
+                            ' in your application use the first option otherwise use the second option.'
+                        ],
+                    }),
+
+
+                    importExample(
+                        `import { Icons } from 'wix-ui-tpa';
+                        ...
+                        render() {
+                          ...
+                          return (
+                            ...  
+                            <Icons.Calendar />,
+                            ...
+                          );
+                        }`
+                    ),
+
+                    importExample(
+                        `import { ReactComponent as Calendar } from 'wix-ui-tpa/dist/src/assets/icons/Calendar.svg';
+                        ...
+                        render() {
+                          ...
+                          return (
+                            ...  
+                            <Calendar />,
+                            ...
+                          );
+                        }`
+                    ),
+
+                    divider(),
+                    title('General'),
+                    code({ title: '', source: `<div style={{display: "grid", gridTemplateColumns: "1fr 1fr 1fr 1fr 1fr 1fr"}}>
+                        <div style={{display: "flex",flexDirection: "column", alignItems: "center", margin: "5px"}}>
+                            <div style={{display: "flex", flexDirection: "column", alignItems: "center", justifyContent: "center", height: " 125px", width: "125px", backgroundColor: "#e6e6e6"}}>
+                                <Anonymous />
```

Comment:
No, that's a different set of icons, the `iconsScope` are icons from `wix-ui-icons-common` which we shouldn't even be using.
I don't see the point for this, why do we need to prefix icons?

Reply:
I agree that all our icons should have the Icon. prefix but currently, the "iconScope" which is "wix-ui-icons-common" is exposed there. Do you mean to expose both of them in this prefix? 
I wonder if changing the "wix-ui-icons-common" prefix will affect anyone.

Reply:
1. AFAIR you said that `wix-ui-icon-common` shouldn't be used, that's why I suggested removing it.
1. Once you will have a `Date` icon, it will collide with the global `Date` object. that's why we always prefix icons.

Reply:
@sivanharel you can change `allComopnents.tsx` to:
```ts
export const allComponents = {
  ...componentsScope,
  ...icons,
  Modal,
  Icons: { ...iconsScope, ...icons },
  ExampleWithContextProps: ({ children, mobile, rtl }) => (
    <componentsScope.TPAComponentsProvider value={{ mobile, rtl }}>
      {children}
    </componentsScope.TPAComponentsProvider>
  ),
  MobileExample,
};
```
and you can remove the `iconsScope` if it doesn't cause too much changes.
But it's not critical for this specific PR.

Regarding the `Icons` prefix, to me it feels a bit confusing, from a documentation point of view, it can confuse consumers on how Icons are used. And name conflicts can be dealt with individually.
But that's only how I feel, and again, it's not critical for this PR IMO